### PR TITLE
Split manifest for prod vs staging so that we can specify the domain for prod

### PR DESCRIPTION
**Pull request by aeijdenberg**
Enable tailing of app during deployment, this is useful for debugging any deploy errors.

### CODE REVIEW

**Comment by aeijdenberg** [comment]
Note since this is a CI change - it's a bit hard to test locally, so we may have to just keep an eye on it in staging...

File: manifest-production.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+inherit: manifest-base.yml
+
+domain: dashboard.gov.au
```
Comment (micapam) [inline manifest-production.yml]: Shouldn't this domain be `smart-cities.dashboard.gov.au` ?

**Review by micapam — CHANGES_REQUESTED** [review]

File: manifest-production.yml (hunk 1 above)
Comment (aeijdenberg) [reply manifest-production.yml]: One would think so, but no, it actually is the parent domain, where the app name ends up sitting as a prefix underneath.

Confusing right? :)

Comment (micapam) [reply manifest-production.yml]: Ah I see, thanks. Maybe drop in a quick comment to that effect in the file?

Comment (aeijdenberg) [reply manifest-production.yml]: Done

**Review by micapam — APPROVED** [review]

**Pull request merged**